addons/im_livechat/static/tests/tours/im_livechat_session_history_open.js

```diff
@@ -0,0 +1,44 @@
+import { registry } from "@web/core/registry";
+
+registry.category("web_tour.tours").add("im_livechat_session_history_open", {
+    steps: () => [
+        {
+            trigger: "body",
+            run: "press ctrl+k",
+        },
+        {
+            trigger: ".o_command_palette_search input",
+            run: "fill /",
+        },
+        {
+            trigger: ".o_command_palette_search input",
+            run: "fill Live Chat",
+        },
+        {
+            trigger: ".o_command:contains(Sessions History)",
```

Comment:
We should click on the full name "Live Chat / Report / Sessions History" if possible, to avoid any possible miss-click if other elements are named the same.